working_groups/learning_journey.md

```diff
@@ -0,0 +1,81 @@
+## Goals
+The primary goal of the Learning Journey Working Group is to provide Nix users with adequate resources to learn Nix on their own. This means that it is the job of the WG to:
+- Ensure that adequate documentation exists
+- Ensure that the documentation is structured in an intuitive way
+- Ensure that the documentation is discoverable
+- Ensure that the documentation meets a minimum standard of quality
+- Establish guidelines for contributions
+
+By extension the WG will be responsible for practical aspects such as:
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- TBD
+
+Membership is open to anyone that wants to contribute.
+
+## Meetings
+Meetings will be held every two weeks to ensure that progress can be made between meetings.
+
+## Immediate projects
+### Decide on documentation structure
+This project will decide on a high level structure for the documentation so that contributors know the audience they are speaking to and where their contributions fit into the Nix documentation landscape.
+
+The documentation will follow the [Diataxis](https://diataxis.fr) framework to the degree that it makes sense. This project will produce a proposal to be submitted to the Documentation Team.
+
+The first draft of this structure is as follows:
+- Tutorials
+    - Installation of Nix
+    - Walking a user through their first derivation
+    - Packaging an existing project for Nix for the first time
+- How-To
+    - Packaging for specific languages
+    - Packaging idioms
+        - `callPackage`, "import from derivation", "fixed output derivation", etc
+    - Day-to-day development and workflows
+        - Integration with IDEs/editors
+        - Nix in CI
+        - Nix and NixOS deployments
+        - Building containers
+    - Rollbacks
+    - Contributing in various ways
+- Explanation
+    - Concepts
+    - What is a derivation?
+    - Cross-compilation as a first-class citizen
+        - Where to put your dependencies (`buildInputs`, `nativeBuildInputs`, etc)
+    - The Nix Store and the database
+    - Channels, profiles, etc
+    - Overlays, overrides, `follows`
+- Reference
+    - Command reference
+    - How store paths are computed
+    - Build phases
+    - How derivations work
+    - Nix language reference
+    - Flake schema
```

Comment:
We did some [shuffling of topics and problems](https://pad.lassul.us/9f-dN2FyScmJ_OOMqAr_ew?both#) with @olaf yesterday in preparation for https://github.com/NixOS/nix.dev/issues/458 and should synchronise all that in the team meetings this week, and see which parts are most meaningful to get funding for.

Related: https://github.com/NixOS/nix.dev/pull/265